JapaneseFilter/sections/specific.txt

```diff
@@ -38,6 +38,7 @@ pokemon-gsc.g-takumi.com,1rk.net,e-typing.ne.jp,zukai-kikenbutu.com,gingerweb.jp
 otanew.jp###extra > div.column-inner > div.column-inner-2 > div.plugin-memo:last-of-type
 nlab.itmedia.co.jp##.bannar
 nlab.itmedia.co.jp##.headerbannar
+energy.rakuten.co.jp##.banner--layout--wrapper
```

Comment:
According to the previous report, there is a leftover `.banner--layout--wrapper` and you block `.banner--layout--wrapper`.
I suggest doing a one common rule `energy.rakuten.co.jp##div[class^="banner--layout"]` will be good for current code or `energy.rakuten.co.jp##div[class^="banner-"]`.

Reply:
Btw, second rule will be good for other sites with a similar banner.